contrib/mesos/pkg/queue/historical.go

```diff
@@ -377,6 +377,16 @@ func (f *HistoricalFIFO) merge(id string, obj UniqueCopyable) (notifications []E
 	return
 }
 
+// Resync will touch all objects to put them into the processing queue
+func (f *HistoricalFIFO) Resync() error {
+	for _, entry := range f.items {
+		if err := f.Update(entry); err != nil {
```

Comment:
I'm not 100% sure it should be Update.
@jdef @sttts - any comments on this line?